apps/judicial-system/web/src/components/PageLayout/utils/index.ts

```diff
@@ -109,6 +110,12 @@ export const getSections = (
           workingCase || ({} as Case),
           activeSubSection,
         ),
-    getCourtSections(workingCase || ({} as Case), user, activeSubSection),
+    isRestrictionCase(workingCase?.type)
+      ? getCourtSections(workingCase || ({} as Case), user, activeSubSection)
```

Comment:
Do we not need a special getExtensionCourtSection for restriction cases?

Reply:
No we don't, because the routing is identical whether it is an extension case or not. You do however point out a flaw in this pr which is that we don't need a special "extension" section for investigation cases either. I've removed it and use "getInvestigationCaseCourtSections" instead.